--- license_map.rb      2008-09-09 18:11:02.000000000 +0100
+++ license_map-i9.rb   2009-04-16 15:05:46.000000000 +0100
@@ -143,6 +143,11 @@
                        "Historical Permission Notice and Disclaimer",
                        /\bhistorical\s*permission\s*notice\s*and\s*disclaimer\b/i ),

+                       SoftwareLicense.new(:i9_license,
+                       "http://i9os.googlecode.com/svn/trunk/Documentation/Licenses/i9_License",
+                       "i9 License",
+                       /\bi9\s*\s*license\b/i  ),
+
                        SoftwareLicense.new(:ibm_public,
                        "http://www.opensource.org/licenses/ibmpl.php",
                        "IBM Public License",

